src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -22,10 +22,20 @@
  */
 public class BoxAPIConnection {
     /**
-     * The default maximum number of times an API request will be tried when an error occurs.
+     * The default total maximum number of times an API request will be tried when error responses
+     * are received.
+     * @deprecated DEFAULT_MAX_RETRIES is preferred because it more clearly sets the number
+     * of times a request should be retried after an error response is received.
      */
+    @Deprecated
     public static final int DEFAULT_MAX_ATTEMPTS = 5;
```

Comment:
Leaving this in doesn't do anything because we don't use this anywhere. By deprecating it, we allow them to set it, but it doesn't affect functionality. So in the end, it is a breaking change unless we find a way to read this value.

Reply:
Correct, it's no longer used in the SDK.  It's deprecated because it's public, so it's possible that there are client apps out there reading this value from the SDK, for use in their client code.  

I don't understand your last sentence.

Reply:
You're right. Forgot it's static and can't be set.